backend/build.gradle.kts

```diff
@@ -25,6 +25,8 @@ repositories {
 
 dependencies {
     implementation("com.google.protobuf:protobuf-kotlin:3.25.2")
+    // can probably be removed when upgrading protoc
+    // https://github.com/protocolbuffers/protobuf/issues/12056
```

Comment:
After updating protobuf I didn't need the additional dependencies. But as updating protoc is not in scope of this I'll readd the dependency again.